src/main/java/seedu/address/model/task/Description.java

```diff
@@ -23,9 +23,10 @@
      * otherwise " " (a blank string) becomes a valid input.
      */
     public static final String VALIDATION_REGEX = "[\\p{Alnum},.!?][\\p{Alnum},.!? ]*";
-    private static final String DEFAULT_DESCRIPTION = "";
```

Comment:
why cant the default desc be an empty string?

Reply:
is it necessary to add an extra boolean when there was nothing wrong with the prev implementation?